data/json/items/comestibles/serums.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "id": "serum_regen",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "category": "drugs",
+    "name": { "str_sp": "unknown liquid" },
+    "description": "White, semi-transparent liquid of unknown origin.",
+    "weight": "7 g",
+    "volume": "10 ml",
+    "charges": 1,
+    "phase": "liquid",
```

Comment:
```suggestion
    "phase": "liquid",
    "container": "test_tube",
```

Reply:
Wait, I found a new bug. If you have this test tube on avatar but don't have a syringe, the content will disapear when you try to use it. Having the test tube on the ground is ok though.
I don't know how to solve it with EOC😹Maybe we should define it as a whole, not liquid and a container separately, to prevent this?

Reply:
Just spawning a new instance if can't be used should handle it; if it doesn't work, i'll make a single unit, yes